ChaosMod/Effects/db/Player/PlayerPacifist.cpp

```diff
@@ -2,17 +2,50 @@
 //Effect by ProfessorBiddle, requested thousands of times on discord
 static void OnTick()
 {
-	Player player = PLAYER_ID();
+	Ped playerPed = PLAYER_PED_ID();
 
-	Entity target;
+	Hash playerHash = GET_ENTITY_MODEL(playerPed);
+	Hash hitHash;
+	int playerHits, playerHitsNew;
 
-	STOP_GAMEPLAY_CAM_SHAKING(true);
+	//get current hits
+	switch (playerHash)
+	{
+	case 225514697: // Michael 
+		hitHash = GET_HASH_KEY("SP0_HITS");
+		break;
+	case 2602752943: // Franklin
+		hitHash = GET_HASH_KEY("SP0_HITS");
+		break;
+	case 2608926626: // Trevor
+		hitHash = GET_HASH_KEY("SP0_HITS");
```

Comment:
You are checking for the same stats as above here

Reply:
Whoops, didn't test with the other two characters 📦 
I'll fix this now.